### PR TITLE
NR-164383 feat: cleanup on windows

### PR DESCRIPTION
**Pull request by rubenruizdegauna**
Cleanup files, packages and services in Windows

### CODE REVIEW

File: caos.ansible_roles/roles/cleanup/tasks/files-Win32NT.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+
+- name: remove infra-agent files and directories
```
Comment (rogercoll) [inline caos.ansible_roles/roles/cleanup/tasks/files-Win32NT.yaml:3]: Is this tied to the infra-agent? Should we change the name?

File: caos.ansible_roles/roles/cleanup/tasks/package-Windows.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Get infra agent productID
+  ansible.windows.win_powershell:
+    script: (Get-WmiObject win32_product | where{$_.name -eq "{{ item }}"}).IdentifyingNumber
+  register: packages_info
+  loop: "{{ packages_to_cleanup }}"
+
+- name: Uninstall infra agent
```
Comment (rogercoll) [inline caos.ansible_roles/roles/cleanup/tasks/package-Windows.yaml:9]:
```suggestion
- name: Uninstall package
```

?

File: caos.ansible_roles/roles/cleanup/tasks/files-Win32NT.yaml (hunk 1 above)
Comment (rubenruizdegauna) [reply caos.ansible_roles/roles/cleanup/tasks/files-Win32NT.yaml]: Nice 👁️ Done in d832c855867ff293db534e296aed06c9009ef371

File: caos.ansible_roles/roles/cleanup/tasks/package-Windows.yaml (hunk 2 above)
Comment (rubenruizdegauna) [reply caos.ansible_roles/roles/cleanup/tasks/package-Windows.yaml]: Nice 👁️ Done in d832c855867ff293db534e296aed06c9009ef371

**Review by rogercoll — APPROVED** [review]
LGTM

**Pull request merged**